perf/trace.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2021 IBM
+# Author: Kalpana Shetty <kalshett@in.ibm.com>
+#
+
+import os
+
+from avocado import Test
+from avocado.utils import distro, process, archive, git
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Linuxtrace(Test):
+
+    """
+    Linux Trace tests
+    """
+
+    failed_tests = list()
+
+    def run_cmd_out(self, cmd):
+        logfile = ("%s/ftracetest.log" % self.teststmpdir)
+        if process.system(cmd, shell=True):
```

Comment:
`ignore_status` is still required. otherwise the test still errors out.

Reply:
Take care of it.